tests/console/install_single_package.pm

```diff
@@ -36,6 +35,10 @@ sub run() {
 
     # ensure that package was installed correctly
     assert_script_run("rpm -q $pkgname");
+    assert_script_run("ci-openqa-tests");
```

Comment:
1. Currently this module will be used ONLY by HPC and it created for HPC so don't see any issue with that. Let's not over engineer things first step is to make it work for HPC later when someone will really want to use it we can easily refactor it 
2. "ci-openqa-tests" is actually designed to be generic and not HPC specific. Any tests package which obey provided contract ( put unit-test script into specified directory) will be able to use this infra
3. Your PR moving totally different idea , not saying that it's wrong but to follow it I need another round of meetings with Egbert while this one is already agreed and because of [1] and [2] don't see point why we need to change something right now